SPV word: New edit meeting form.

jone opened the pull request:
**Background:**
In the older SPV implementation, the meeting details were edited through the "protocol"-view, which included the meeting details form.
But with the word feature we are no longer using the "protocol"-view because the protocol is now written directly in the word files, triggered through the meeting view.
Request: https://github.com/4teamwork/gever/issues/65

**Changes:**
- The meeting details form is extracted from the "protocol"-view into a separate edit form.
- The "edit"-action is moved from the left sidebar to Plone's editbar (`object`-action), because it is the plan to get rid of the sidebar step by step.

**Implementation details:**
- The meeting-type has no `FTI` => the action can only be registred publicly.
- The published meeting wrapper does not allow unsecure access to attributes, therefore the edit-condition-check is implemented on the view.
- The fixture is updated to define explicit membership periods, so that the members get available in the edit form.
- A `deactivate_feature`-method is added to the test case base class. It makes it easier to test action availability.
- Locking and conflict detection is copied along from the protocol view.
- Because of consistency, the look-and-feel should go towards Plone / rest of GEVER.
- The status button in the sidebar is now visually broken. It will be moved to a separate place and thus fixing is not worth the effort.

![bildschirmfoto 2017-09-18 um 11 07 54](https://user-images.githubusercontent.com/7469/30535501-e4f5f2d6-9c62-11e7-9298-0c6ea0335376.png)
![bildschirmfoto 2017-09-18 um 11 07 49](https://user-images.githubusercontent.com/7469/30535502-e4ff1ff0-9c62-11e7-92bf-17a721abed8d.png)

jone commented:
Rebased because of changelog conflict.

deiferni commented on opengever/meeting/browser/meetings/edit_meeting.py:
```diff
@@ -0,0 +1,157 @@
+from zExceptions import Redirect
+
+
+class EditMeetingView(ModelEditForm):
```
please add a short docstring here mentioning that this is currently a copy of the protocol edit class and why

jone commented:
Updated according to your suggestions.
(Be aware that I've renamed the object-action according to the renamed view. If you did run the upgrade step from the old implementation you'll end up with a wrong action `edit_meeting` and a correct action `edit-meeting`.)

deiferni commented on opengever/meeting/browser/meetings/edit_meeting.py:
```diff
@@ -0,0 +1,164 @@
+class EditMeetingView(ModelEditForm):
+    """The edit meeting view provides a form for editing the meeting details.
+
+    This view is used when the edit-meeting feature is enabled.
```
edit-meeting -> word meeting?

jone commented:
Updated 👍
